search_gov_crawler/search_gov_spiders/settings.py

```diff
@@ -111,3 +116,33 @@
     "http": "scrapy_playwright.handler.ScrapyPlaywrightDownloadHandler",
     "https": "scrapy_playwright.handler.ScrapyPlaywrightDownloadHandler",
 }
+
+now = datetime.now()
+date_time = now.today().isoformat()
+dirname= os.path.dirname(__file__)
+body_html_template = os.path.join(dirname, 'actions', 'results.jinja')
+
+SPIDERMON_ENABLED = os.environ.get('SPIDERMON_ENABLED')
```

Comment:
Even though scrapy's get_bool function https://github.com/scrapy/scrapy/blob/212e848402a63b43fe8b7204e19d47fa7c4f0cd9/scrapy/settings/__init__.py#L126 allows this to be really flexible, I think it would be more clear to add a default so that someone can understand that this is meant to be a boolean: `os.environ.get('SPIDERMON_ENABLED', 'False')` for example.  What do you think?